components/home/modules/ROOT/pages/index.adoc

```diff
@@ -1,64 +1,86 @@
 
-Gas Station Network is a service that you can integrate in your dapp to abstract all transaction and gas costs.
+=== Ethernaut
 
-More information about Gas Station Network:
+Learn more about security in Solidity contracts by playing https://ethernaut.openzeppelin.com/[Ethernaut]. Every level in Ethernaut is a smart contract waiting to be hacked, and teaches you a vulnerability by having you exploit it - many of them based on real-life hacks! You can discuss your solutions or ask for tips in https://forum.openzeppelin.com/c/security/ethernaut/30[our forum].
 
-https://gsn.openzeppelin.com/[Website].
-https://medium.com/@rrecuero/eth-onboarding-solution-90607fb81380[Announcement]
+=== Careers
 
-=== Components
-In order to integrate Gas Station Network into your application, you need to slightly modify your contracts with OpenZeppelin library and then connect to a new provider via web3.
+The OpenZeppelin https://openzeppelin.com/about/[team] is always hiring. If you have enjoyed using our tools or reading our audit reports, and want to be a part of it, check out our https://openzeppelin.com/jobs/[current job openings] and join us!
 
-* *Contracts*: See how to use OpenZeppelin Library to integrate GSN https://docs.openzeppelin.com/contracts/2.x/gsn[here].
-* *Fund your dapp recipient*: Use the dapp tool to fund your application https://gsn.openzeppelin.com/recipients[here].
-* *Web3 Provider*: Check how to use the GSN web3 provider https://github.com/OpenZeppelin/openzeppelin-gsn-provider[here].
-  We recommend using https://github.com/OpenZeppelin/openzeppelin-network.js[OpenZeppelin Network.js] to handle different Web3 Providers.
-* *Testing GSN*: Test and development helper methods and scripts for using GSN are available https://github.com/OpenZeppelin/openzeppelin-gsn-helpers[here].
+=== Social
 
-=== GSN Sample App: Online Chat
-We have created a sample chat application to showcase the power of gasless dapps. As you can see you don't need Metamask or any other wallet to interact with it.
-You can take a look and the code and see an example of how to integrate GSN using https://github.com/OpenZeppelin/openzeppelin-network.js[OpenZeppelin Network.js] in the https://github.com/OpenZeppelin/gsn-sample-chat_app/blob/master/client/src/App.js[sample app].
+// We may want to remove this if it's going to be present on the footer?
+You can https://twitter.com/OpenZeppelin[follow us on Twitter], check out https://blog.openzeppelin.com/[our Blog], tune in https://www.youtube.com/channel/UC0Nx8RbugiCzdlVGK7hWZfw/videos[our YouTube channel], or join https://forum.openzeppelin.com/[our Forum]!
```

Comment:
I'd bump the forum and blog (in that order), perhaps dropping twitter and youtube. However, given the footer with the 'ask for help' call to action (will those be on every page?), we could indeed remove this whole thing.

Reply:
Agree on removing. I'm just reordering for now until we decide if we drop it or not.